pootle/apps/pootle_app/management/commands/update_stores.py

```diff
@@ -44,9 +44,9 @@ def handle_translation_project(self, translation_project, **options):
         :return: flag if child stores should be updated
         """
         if translation_project.directory_exists_on_disk():
-            logging.info(u"Scanning for new files in %s", translation_project)
-            translation_project.scan_files()
-            return True
+            translation_project.update_from_disk(
+                force=options['force'], overwrite=options['overwrite'])
+            return False
```

Comment:
`translation_project.update_from_disk()` will do everything, `False` means that it `PootleCommand` won't trigger any of store handlers.